EIPS/EIP-961.md

```diff
@@ -0,0 +1,58 @@
+---
+    eip: 961
+    title: URL Format for Tokens
+    status: Active
+    type: Standards Track
+    author: Kylan Hurt <kylan@edgesecure.co>, Eliran Zach <eliran@edgesecure.co>
+    created: 2018-04-01
+---
+
+## Simple Summary
+A standard way of representing token information, facilitating the process of adding ERC #20 tokens as URLs on the application layer.
+
+## Abstract
+URLs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very loosely coupled applications. A standardized URL format for fundamental token information allows for instant invocation of the user's preferred wallet application (even if it is a webapp or a swarm đapp), with the correct parameterization of the token's information only to be confirmed by the user.
+
+## Motivation
+The convenience of representing payment requests by standard URLs has been a significant factor in the cryptocurrency. Extending this convenient mechanism to support embedding of basic token information would help to facilitate the usage of Ethereum-based tokens. In particular, QR-code embedded URLs could be generated by token-generators to help users track the tokens via any wallet software that has the ability to scan QR codes (particularly mobile wallets). For example, rather than users needing to find the contract address and number of decimal places for a given token through an online directory, they could simply scan a QR-code provided by the token generator with this data embedded. The wallet software could then automatically add the tokens to the user's wallet. This would improve the user experience associated with purchasing and holding tokens as a digital asset.
+
+
+## Specification
+
+### Syntax
+Token data URLs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+
+    request                 = "ethereum" ":" "token_info-" target_address [ "@" chain_id ] [ "?" parameters ]
+    target_address          = ethereum_address
+    chain_id                = 1*DIGIT
+    ethereum_address        = ( "0x" 40*40HEXDIG ) / ENS_NAME
+    parameters              = parameter *( "&" parameter )
+    parameter               = key "=" value
+    key                     = "symbol" / "decimals" / "name" / "type"
+    value                   = STRING / number (0-18) / STRING / STRING
+
+Where all of the key + value pairs are optional, allowing for maximum flexibility on the application layer.
+
+
+### Semantics
+
+`target_address` is mandatory and denotes either the beneficiary of native token payment (see below) or the contract address with which the user is asked to interact.
```

Comment:
what does "see below" refer to? As far as I see this should always be the "contract address" of the token

Reply:
Looks like it could be hold-over from the earlier EIP that we referenced. I will try to make the edit later today. Thank you for the heads up.